<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormChangePassword</class>
 <widget class="QWidget" name="FormChangePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="bgContainer">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-30</y>
     <width>701</width>
     <height>741</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bgContent">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>49</y>
     <width>521</width>
     <height>521</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>421</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	
	padding: 0 8px;

	color:rgb(10,100,180);
	font: 24pt &quot;Showcard Gothic&quot;;
}
</string>
   </property>
   <property name="text">
    <string>Change password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	
	padding: 0 8px;

	color:rgb(10,100,180);
	font: 10pt &quot;Showcard Gothic&quot;;
}
</string>
   </property>
   <property name="text">
    <string>Old password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="oldPassword">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>220</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{


	padding: 0 8px;
	background:white;
	color:rgb(10,100,180);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	
	padding: 0 8px;

	color:rgb(10,100,180);
	font: 10pt &quot;Showcard Gothic&quot;;
}
</string>
   </property>
   <property name="text">
    <string>new passwrod:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="newPassword">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{


	padding: 0 8px;
	background:white;
	color:rgb(10,100,180);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	
	padding: 0 8px;

	color:rgb(10,100,180);
	font: 10pt &quot;Showcard Gothic&quot;;
}
</string>
   </property>
   <property name="text">
    <string>Confirm password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmPassword">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>380</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{


	padding: 0 8px;
	background:white;
	color:rgb(10,100,180);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>460</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 14pt &quot;Showcard Gothic&quot;;
	padding: 0 8px;
	background:white;
	color:rgb(10,100,180);
border:10px;
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>460</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	font: 14pt &quot;Showcard Gothic&quot;;
	padding: 0 8px;
	background:white;
	color:rgb(10,100,180);
border:10px;
}</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHidenNewPassword">
   <property name="geometry">
    <rect>
     <x>511</x>
     <y>304</y>
     <width>24</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background:white;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHidenOldPassword">
   <property name="geometry">
    <rect>
     <x>511</x>
     <y>225</y>
     <width>24</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background:white;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="stataus">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	
	padding: 0 8px;

	color: rgb(191, 0, 0);
	font: 8pt &quot;Showcard Gothic&quot;;

}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="errorOldPassword">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>250</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 6pt &quot;Segoergb(255, 21, 72) UI&quot;;
	
	color: rgb(191, 0, 0);
	
	padding: 0 8px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="errorNewPassword">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 6pt &quot;Segoergb(255, 21, 72) UI&quot;;
	
	color: rgb(191, 0, 0);
	
	padding: 0 8px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="errorConfirmPassword">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>410</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
	font: 6pt &quot;Segoergb(255, 21, 72) UI&quot;;
	
	color: rgb(191, 0, 0);
	
	padding: 0 8px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
